src/components/Comments/Comment.js

```diff
@@ -358,7 +359,7 @@ const styles = StyleSheet.create({
   btnReplyText: {
     fontFamily: fonts.inter[400],
     fontSize: 13,
-    color: COLORS.gray1,
+    color: '#C4C4C4',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been updated. Ensure this change is consistent with the overall design and color scheme of the application.
```diff
- color: COLORS.gray1,
+ color: '#C4C4C4',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->